src/store/JasonYbReduxShoppingList/actions.ts

```diff
@@ -2,7 +2,7 @@ import { Dispatch, GetState } from '../types'
 import { ActionTypes, SHOPPING_LIST_ACTIONS, ShoppingItem } from './types'
 
 export const AddItemQuantity =
```

Comment:
o my.. all these shouldve been camelCase

Reply:
Resolved in 80182f1